vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -26,20 +26,23 @@ class Accordion(VizroBaseModel):
     _validate_pages = validator("pages", allow_reuse=True, always=True)(_validate_pages)
 
     @_log_call
-    def build(self, active_page_id):
+    def build(self, *, active_page_id=None):
         return self._create_accordion(active_page_id=active_page_id)
 
     def _create_accordion_buttons(self, pages, active_page_id):
         """Creates a button for each provided page that is registered."""
         accordion_buttons = []
         for page_id in pages:
-            page = dash.page_registry[page_id]
+            try:
+                page = dash.page_registry[page_id]
+            except KeyError:
+                raise KeyError(f"Page with ID {page_id} cannot be found. Please add the page to `Dashboard.pages`")
```

Comment:
```suggestion
            except KeyError as exc:
                raise KeyError(f"Page with ID {page_id} cannot be found. Please add the page to `Dashboard.pages`") from exc
```

Very minor, but it's generally nice to chain these sort of exceptions.